extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -228,22 +228,22 @@ export async function getCompletionItemsFromSpecs(
 		if (result.items) {
 			items.push(...result.items);
 		}
-	}
-
-	if (tokenType === TokenType.Command) {
-		// Include builitin/available commands in the results
-		const labels = new Set(items.map((i) => i.label));
-		for (const command of availableCommands) {
-			if (!labels.has(command.label)) {
-				items.push(createCompletionItem(terminalContext.cursorPosition, prefix, command, command.detail));
+	} else {
```

Comment:
I think we want it how it was before? Otherwise, it's possible that result had no items, neither files nor folders were requested, and we won't change that.

Reply:
Good call, that's why the tests failed too 👍